test/parallel/test-runner-concurrency.js

```diff
@@ -62,3 +69,14 @@ describe(
     it('should run after other suites', expectedTestTree);
   });
 }
+
+test('--test multiple files', { skip: os.availableParallelism() < 3 }, async () => {
```

Comment:
@cjihrig please confirm this skip makes sense

Reply:
I think it makes sense. I think you could verify by replacing the use of `os.availableParallelism()` in the test runner code with 1 or 2 and seeing if the test hangs.